### DISCUSSION TITLE
elementaryos-5.1-stable.20200204 - No wifi after system update

### DISCUSSION THREAD

**Discussion by todor-kolev** [elementary/switchboard-plug-network#370] [Q&A]
The system doesn't allow you to connect to wifi and there's no wifi icon at all. This happened after installing the system update from the App centre in elementaryos-5.1-stable.20200204. Everything was working fine before I installed the system update and that was the only update I installed. I also tried installing the latest version (at the time) elementaryos-5.1-stable.20200405 from scratch and it doesn't even prompt me to connect to the wifi during installation. After installation completed there was no wifi icon in the notification area. 

This is on Lenovo ideapad c340
  *-network                 
       description: Wireless interface
       product: Intel Corporation
       vendor: Intel Corporation
       physical id: 14.3

**Comment by cassidyjames** [comment]
Did you also install a WiFi driver listed in AppCenter?

**Comment by todor-kolev** [comment]
So I tried upgrading with and without installing the driver. Installing the driver on its own did not break the wifi but upgrading the system did (with and without the network driver installed).
